src/State.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Clue\React\Mq;
+final class State
```

Comment:
```suggestion
namespace Clue\React\Mq;

/** @internal */
final class State
```

Just added some white space and I think we should mark this class as internal